pkg/referrerstore/oras/oras.go

```diff
@@ -111,10 +112,18 @@ func (s *orasStoreFactory) Create(version string, storeConfig config.StorePlugin
 		return nil, fmt.Errorf("could not create local oras cache at path %s: %s", conf.LocalCachePath, err)
 	}
 
+	insecureHttp := http.DefaultClient
```

Comment:
according to this post, looks like `http.Client` is preferred. https://www.loginradius.com/blog/engineering/tune-the-go-http-client-for-high-performance/

Reply:
yeah I've tried the settings mentioned here before. Doesn't seem to alleviate the performance degradation caused by 100+ artifacts pulling at once. I'll update the PR to use these anyways.